api/src/pcapi/routes/backoffice/venues/forms.py

```diff
@@ -132,7 +133,8 @@ def validate_acceslibre_url(self, acceslibre_url: fields.PCOptStringField) -> fi
             raise validators.ValidationError(
                 "L'URL doit se terminer par /<slug-chez-acceslibre>/ (un slug n'est composé que de minuscules et de tirets du milieu)"
             )
-
+        if not offerers_api.id_exists_at_accessibility_provider(slug=acceslibre_url.data.split("/")[-2]):
+            raise validators.ValidationError("Cet URL n'existe pas chez acceslibre")
```

Comment:
Le Larousse et le Robert sont d'accord pour dire qu'URL est un nom féminin : 
```suggestion
            raise validators.ValidationError("Cette URL n'existe pas chez acceslibre")
```
Sources : 
https://www.larousse.fr/dictionnaires/francais/URL/80723
https://dictionnaire.lerobert.com/definition/url